addons/rose/addon/components/rose/filterable-dropdown/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { debounce } from 'core/decorators/debounce';
+
+// TODO: Remove old Rose::Dropdown and rename this to Rose::Dropdown
+export default class RoseFilterableDropdownComponent extends Component {
+  // =attributes
+
+  @tracked selectedItems = [];
+  @tracked searchTerm;
+
+  // =methods
+
+  /**
+   * Takes itemOptions and filters them by searchTerm if there is one
+   * @returns {[object]}
+   */
+  get itemOptions() {
```

Comment:
It doesn't matter too much but this shouldn't be under the methods section you labeled on line 18

Reply:
Fixed.